drivers/adc/ad7606/ad7606.c

```diff
@@ -600,6 +602,71 @@ int32_t ad7606_reg_write_mask(struct ad7606_dev *dev,
 	return ad7606_reg_write(dev, addr, reg_data);
 }
 
+/***************************************************************************
+ * @brief Correction bit31 - bit18 for negative value (for 18 bits ADC7606 data via spi)
+ *
+ * @param dev       - The device structure pointer
+ * @param pdata     - data buffer pointer
+ * @param size      - Number of data needs correct
+ * @return 
+ *******************************************************************************/
+static void adcdata_sign_correction(struct ad7606_dev* dev, uint32_t size, uint32_t* pdata)
```

Comment:
The code indentation looks messed up. Please use 8 spaces. Or just run `ci/astyle.sh`. It should also fix the other style issue I've pointed.